particles/TestingExample/ComputePeopleStatsTest.kt

```diff
@@ -0,0 +1,44 @@
+package arcs.unit.test.example
+
+import arcs.sdk.testing.TestingCollection
+import arcs.sdk.testing.TestingSingleton
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/**
+ * Example of particle Unit Testing.
+ */
+@ExperimentalCoroutinesApi
+@RunWith(JUnit4::class)
+class ComputePeopleStatsTest {
+
+    @Test
+    fun testMedianReactsToInputChanges() = runBlockingTest {
```

Comment:
nit: `test` prefix in method names is not needed and AFAIK usually omitted style-wise with JUnit4 where `@Test` annotation is sufficient.

Reply:
Thanks, done :)